File: app/src/main/java/com/airfreshener/telegram_sms/model/RequestMessage.java
package com.airfreshener.telegram_sms.model;

public class RequestMessage {
    //Turn off page preview to avoid being tracked
    @SuppressWarnings({"unused", "RedundantSuppression"})
    public final boolean disable_web_page_preview = true;
    public long message_id;
    public String parse_mode;
    public String chat_id;
    public String text;
    public ReplyMarkupKeyboard.keyboard_markup reply_markup;
}

File: app/src/main/java/com/airfreshener/telegram_sms/model/RequestMessage.kt
package com.airfreshener.telegram_sms.model

import com.airfreshener.telegram_sms.model.ReplyMarkupKeyboard.KeyboardMarkup

@Suppress("unused", "PropertyName")
class RequestMessage {

    val disable_web_page_preview = true //Turn off page preview to avoid being tracked
    var message_id: Long = 0
    var parse_mode: String? = null
    var chat_id: String? = null
    var text: String? = null
    var reply_markup: KeyboardMarkup? = null
}
